src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String ownerName;
+    private int integerVariable;
```

Comment:
The variable's name should reflect the content of what's inside it. Please try to choose more appropriate names for these two variables.